packages/verification-app/.env

```diff
@@ -7,4 +7,4 @@ VUE_APP_CLAIM_CONTRACT_ADDRESS=
 VUE_APP_ACCEPTED_ISSUER_KEY=
 VUE_APP_CERTIFACTION_API_URL=
 VUE_APP_CERTIFACTION_CDN_BASE_URL=
-VUE_APP_PDF_WASM_VERSION=2.1.11
+VUE_APP_PDF_WASM_VERSION=2.2.3
```

Comment:
2.3.0 would be the most recent version. The change to that version was merged on certifaction-web yesterday.

Reply:
Updated